quarkchain/cluster/shard.py

```diff
@@ -411,6 +417,13 @@ def __validate_block_headers(self, block_header_list: List[MinorBlockHeader]):
                 return False
         return True
 
+    def _qkchashX_enabled(self, header):
```

Comment:
nit: rename `_qkchashX` to `_qkchashx` since variable name should all be lower case